app.js

```diff
@@ -1,6 +1,7 @@
 const express = require('express');
 
 const hbs = require('hbs');
+const { get } = require('http');
```

Comment:
Here you're importing the get from `http` but you are not using anywhere actually, if you take a look at your code it's not marked as been used and if you comment the line the pages keep working